src/Model/CampaignCategory.php

```diff
@@ -62,9 +62,9 @@ public function addCategory(CampaignCategory $category)
     /**
      * @return CampaignCategory[]
      */
-    public function getCategories(): array
+    public function getCategories()
     {
-        return $this->categories;
+        return $this->categories ?? [];
```

Comment:
You can leave the return type restriction here if you force to always return an array. We can also set the default array type in the private property:

```
$this->categories = [];
```

Reply:
Yes that is true